logging/performance_reliability/logging-content-filtering.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-content-filtering"]
+= Filtering logs by content
+:context: logging-content-filtering
+
+toc::[]
+
+Collecting all logs from a cluster may produce a large amount of data, which can be expensive to transport and store.
+
+You can reduce the volume of your log data by filtering out low-value data that does not need to be stored. {logging} provides content filters that can be used to reduce the volume of log data.
```

Comment:
The 2nd sentence begins with a lower case "logging".

Reply:
also may want to use the term "low-priority" rather than low-value, although both are subjective :smile:

Reply:
I would say "low-value" makes more sense, since it sort of explains _why_ something would be considered low-priority 🙂

Reply:
nevermind, updated to low priority to match the other module